router/dataplane.go

```diff
@@ -973,14 +982,14 @@ type scionPacketProcessor struct {
 	// For a hop performing an Xover, it is the MAC corresponding to the down segment.
 	cachedMac []byte
 	// macBuffers avoid allocating memory during processing.
-	macBuffers macBuffers
+	macBuffers macBuffersT
 
 	// bfdLayer is reusable buffer for parsing BFD messages
 	bfdLayer layers.BFD
 }
 
-// macBuffers are preallocated buffers for the in- and outputs of MAC functions.
-type macBuffers struct {
+// macBuffersT are preallocated buffers for the in- and outputs of MAC functions.
+type macBuffersT struct {
```

Comment:
Yeah, I have added the comment but not pushed yet because I am still fixing other things and don't want to make the thing run or just a comment addition. I should have resolved the thing after I pushed though.

Reply:
sorry, then it was a misunderstanding from my side